### PR TITLE
Rebuild 9.0.1

### PR DESCRIPTION
**Pull request by basnijholt**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

File: .ci_support/linux_64_build_varianteglpython3.8.____cpython.yaml [hunk 1]
```diff
@@ -85,13 +59,13 @@ sqlite:
...
-- '2020'
+- '2021'
 tbb_devel:
-- '2020'
+- '2021'
 tk:
 - '8.6'
 vtk:
-- 9.0.1
+- 9.1.0
```
Comment (basnijholt) [inline .ci_support/linux_64_build_varianteglpython3.8.____cpython.yaml:68]: Why would it pin vtk here?

Comment (Tobias-Fischer) [reply .ci_support/linux_64_build_varianteglpython3.8.____cpython.yaml:68]: I'm not sure why this happens - vtk is pinned in conda-forge, but it should not pin vtk itself in the vtk feedstock. Anyhow, it should not cause any issues in the build process.

**Comment by basnijholt** [comment]
@ conda-forge-admin, please rerender

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
Hi @basnijholt - do you have any plans to continue working on this, or can we close here?

**Pull request closed without merging**